web/src/pages/Upstream/locales/en-US.ts

```diff
@@ -61,10 +68,12 @@ export default {
   'upstream.step.healthy.checks.successes': 'Successes',
   'upstream.step.input.healthy.checks.successes': 'Please input successes',
   'upstream.step.healthy.checks.http_failures': 'HttpFailures',
+  'upstream.step.healthy.checks.active.create.req_headers': 'Creat req_headers',
```

Comment:
Hi @LiteSun, please help @guoqqqi to update those i18n values when you are free 💯